src/content/docs/ai-monitoring/view-ai-data.mdx

```diff
@@ -0,0 +1,187 @@
+
+Your AI response table organizes data about interactions between your end user and AI app. You can view when an interaction occurred, prompts paired with their responses, completion and token count, and which model received a prompt. 
+
+<img
+    title="AI response page response table"
+    alt="A cropped screenshot displaying the response table from the AI responses view"
+    src={aiResponseTable}
+/>
+
+You can adjust the table columns by clicking the cog icon in the upper right. This functionality lets you choose the kinds of data you want to analyze.
+
+The Responses is an entry point into viewing trace data about an individual response. Click a row in the table to open the trace view of a particular response.
+
+### AI response trace view
+
+<img
+    title="AI response trace view"
+    alt="A screenshot of the trace view for a particular AI response"
+    src={aiTraceViewAiResponse}
+/>
+
+The AI response trace view gives you trace-level insights into how your app generates responses. You may want to look at the trace view to identify where an error occurred, or maybe you want to understand what led to negative feedback from a high token response. 
+
+* Choose between traces or logs. When you select logs, query within logs for text strings or attributes you want to investigate further.
+* Toggle between response details or metadata. The response details column shows the user prompt and AI response so you can maintain context for your traces and spans. Metadata provides a list view for entity GUID, model, tokens, and vendor.
+* When an error occurs, the waterfall view highlights its row in red. Select the row to open up span data, including the span's error details.
+
+        <SideBySide>
+            <Side>
+
+            <img
+                title="Span details modal"
+                alt="A screenshot that shows span details "
+                src={aiTraceWaterfallPageSpanDetails}
+            /> 
+
+            </Side>
+            <Side>
+            <img
+                title="Error details modal"
+                alt="A screenshot that shows error details "
+                src={aiTraceWaterfallPageErrorDetails}
+            /> 
+            </Side>
+        </SideBySide>
+
+
+## AI entities page [#entities]
+
+The AI entities page organizes all your entities currently reporting AI data into a table. This page displays your AI apps alongside response time, throughput, and error rate. 
+
+    <img
+        title="AI entities page"
+        alt="A screenshot of the first page you see when you click AI Monitoring. View aggregated data, compare your AI models, or create drop filters."
+        src={aiAIEntitiesPage}
+    />
+<figcaption>
+    View the entities that report AI data: Go to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI Monitoring**
+</figcaption>
+
+Selecting an AI entity takes you to the APM summary page for that app. From the **APM summary page**, select **AI monitoring** in the left nav. 
```

Comment:
This section reads like something is coming after you select AI monitoring. I'd consider restructuring it to be more solid.